pnlscripts/dwiconvert.py

```diff
@@ -27,9 +27,11 @@ class App(cli.Application):
 
     Examples:
         dwiconvert.py -i dwi.nii.gz -o dwi.nrrd
+	dwiconvert.py -i dwi.nii.gz -o dwi.nhdr
```

Comment:
indent

Reply:
Should be fixed, or may be just web artifact!